manifests/crd/acceleratorprofiles.opendatahub.io.crd.yaml

```
@@ -28,27 +28,36 @@ spec:
               properties:
                 displayName:
                   type: string
+                  description: "The display name of the accelerator profile."
                 enabled:
                   type: boolean
+                  description: "Indicates whether the accelerator profile is available for new resources."
                 identifier:
                   type: string
+                  description: "The resource identifier of the accelerator device. Example: nvidia.com/gpu"
                 description:
                   type: string
+                  description: "A short description of the accelerator profile."
                 tolerations:
                   type: array
+                  description: "A Kubernetes toleration value that is added to resources using this accelerator profile."
```

```suggestion
                  description: "Any number of Kubernetes toleration values that are added to resources when created or updated to this accelerator profile."
```
My bad, this is an array. I think it has some issues as it shouldn't talk about:
- A singular instance, it really is "any number" 0 - infinity (I guess?)
- Infer that it's "attached" to the resource -- since changing this after the fact won't impact existing resources.